### PR TITLE
Updated TAP installation flow with  env and comments to make it more readable

### PR DESCRIPTION
**Pull request by kumaritanushree**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
None

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by dhadukk** [comment]
looks good to me.

**Comment by YugangW** [comment]
Hi @kumaritanushree, Would you mind helping us resolve the conflict so we can get it merged?

**Comment by kumaritanushree** [comment]
@YugangW resolved conflict

File: install-online/profile.hbs.md [hunk 1]
```diff
@@ -29,17 +29,28 @@ To relocate images from the VMware Tanzu Network registry to your registry:
...
+    # Note: Above IMGPKG_* envs are being used by imgpkg command only
+
+    # Registry to fetch TAP Packages from
+    export INSTALL_REGISTRY_USERNAME=”${IMPKG_REGISTRY_USERNAME_1}”
+    export INSTALL_REGISTRY_PASSWORD=”${IMPKG_REGISTRY_PASSWORD_1}”
+    export INSTALL_REGISTRY_HOSTNAME=”${IMPKG_REGISTRY_HOSTNAME_1}”
     export TAP_VERSION=VERSION-NUMBER
     export INSTALL_REPO=TARGET-REPOSITORY
+
+    # User’s registry where TAP will store built images and as well as the tanzu build -service dependencies. Ensure these credentials have write permission.
```
Comment (joaopapereira) [inline install-online/profile.hbs.md:50]:
```suggestion
    # User’s registry where TAP will store built images and as well as the tanzu build-service dependencies. Ensure these credentials have write permission.
```

File: install-online/profile.hbs.md [hunk 2]
```diff
@@ -73,16 +84,36 @@ To relocate images from the VMware Tanzu Network registry to your registry:
     imgpkg copy -b registry.tanzu.vmware.com/tanzu-application-platform/tap-packages:${TAP_VERSION} --to-repo ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/tap-packages
     ```
 
-## <a id='add-tap-repo'></a> Add the Tanzu Application Platform package repository
-
-Tanzu CLI packages are available on repositories. Adding the Tanzu Application Platform package repository makes Tanzu Application Platform and its packages available for installation.
+<details>
 
+<summary>If you do not want to relocate images</summary>
 [Relocate images to a registry](#relocate-images) is strongly recommended but not required for installation. If you skip this step, you can use the following values to replace the corresponding variables:
```
Comment (joaopapereira) [inline install-online/profile.hbs.md:90]:
```suggestion
[Relocate images to a registry](#relocate-images) is strongly recommended but not required for installation. If you skip that step, you can execute the following commands in your command line to set the source of the OCI Images to be TanzuNet:
```

File: install-online/profile.hbs.md [hunk 3]
```diff
@@ -73,16 +84,36 @@ To relocate images from the VMware Tanzu Network registry to your registry:
...
-- `TAP_VERSION` is your Tanzu Application Platform version. For example, `{{ vars.tap_version }}`
+```console
+# Registry to fetch TAP Packages from.
+export INSTALL_REGISTRY_USERNAME=TANZUNET_REGISTRY_USERNAME
+export INSTALL_REGISTRY_PASSWORD=TANZUNET_REGISTRY_PASSWORD
+export INSTALL_REGISTRY_HOSTNAME=”registry.tanzu.vmware.com”
+export TAP_VERSION=VERSION-NUMBER
+export INSTALL_REPO="tanzu-application-platform"
+
+# User’s registry where TAP will store built images and as well as the tanzu build -service dependencies. Ensure these credentials have write permission.
```
Comment (joaopapereira) [inline install-online/profile.hbs.md:100]:
```suggestion
# User’s registry where TAP will store built images and the Tanzu build-service dependencies. Ensure these credentials have write permission.
```

**Review by joaopapereira — COMMENTED** [review]
Please create a new PR with the following changes

**Pull request merged**